src/js/index.js

```diff
@@ -44,37 +44,58 @@ RequestUtil.json = function (options = {}) {
   oldJSON(options);
 };
 
-function onApplicationLoad() {
-  // Allow overriding of application contents
-  let contents = PluginSDK.Hooks.applyFilter('applicationContents', null);
-  if (contents) {
-    ReactDOM.render(
-      (<Provider store={PluginSDK.Store}>
-        contents
-      </Provider>),
-      domElement);
-  } else {
-    setTimeout(function () {
-      let routes = RouterUtil.buildRoutes(appRoutes.getRoutes());
-      let router = Router.run(routes, function (Handler, state) {
-        Config.setOverrides(state.query);
-        ReactDOM.render(
-          (<Provider store={PluginSDK.Store}>
-            <Handler state={state} />
-          </Provider>),
-          domElement);
+(function () {
```

Comment:
It just keeps the code that is related to each other together, if you feel strongly about it I can remove it. I just figured it's easy to come back to this in the future and see what is related in this file, since it's just one long thing of stuff.